test-suite/buildtests/layer-02-maximus.opts

```diff
@@ -111,7 +111,7 @@ DISTCHECK_CONFIGURE_FLAGS=" \
 	--with-pthreads \
 	--enable-build-info=squid\ test\ build \
 	--enable-ssl-crtd \
-	--with-openssl \
+	--with-openssl=${with_openssl:-yes} \
```

Comment:
Homebrew refuses to symlink openssl libraries in /usr/local/{lib,include}, insisting instead to symlink in /usr/local/opt/openssl/{include,lib}. This is done deliberately in order to prevent polluting the system environment with non-system-supplied libraries and headers.
The actual underlying issue is that --with-openssl[=location] acts both as an enable switch and as a location pointer (with --with-openssl and --with-openssl=yes being synonyms for "enable, includes and headers are in custom location".
A system-specific workaround is to pass the location in with the $with_openssl environment variable, but maximus will overwrite that to "yes". This hack fixes that by letting the information through.
LIBOPENSSL_CFLAGS and LIBOPENSSL_LIBS seem to be just duplicates --with-openssl.